scipy/stats/tests/test_distributions.py

```diff
@@ -3139,6 +3139,28 @@ def test_cases(self):
         assert_almost_equal(stats.trapz.cdf(0.9, 0.2, 0.8), 0.96875)
         assert_almost_equal(stats.trapz.cdf(1.0, 0.2, 0.8), 1.0)
 
+        # issue #11795: improve precision of trapz stats
```

Comment:
Can you put this whole set of tests as a separate function? e.g. 

```
def test_moments_and_entropy(self):
   <your code here>
``` 
that will separate out the calculations from the `test_cases` (unfortunate naming) test case above.